### ISSUE TITLE
update to newer version of gnome runtime flatpak required

### ISSUE THREAD

**Issue by stigman9393** [NickvisionApps/Cavalier#165]
Upgrade is required org.gnome.Platform, 45 is no more supported:
   The GNOME 45 runtime is no longer supported as of September 18, 2024. Please ask your application developer to migrate to a supported platform.

**Comment by DiMTRX** [comment]
Almost March '25 and just finished a flatpak install and issue is still 'around'. Just thought I should mention it here :)

> Info: runtime org.gnome.Platform branch 45 is end-of-life, with reason:
   The GNOME 45 runtime is no longer supported as of September 18, 2024. Please ask your application developer to migrate to a supported platform.

**Comment by Drexxl-the-Walrus** [comment]
The issue still persists

**Comment by stigman9393** [comment]
> The issue still persists

it looks like maintainer stops to maintain this app and not planning to update this app anymore :(

**Comment by nlogozzo** [comment]
> and not planning to update this app anymore

That's not true. I plan to update it, just when I have time available to do so.

**Comment by schneik80** [comment]
6 months later and no update. Any news?
